core/sdk-core/src/test/java/software/amazon/awssdk/core/exception/SdkExceptionTest.java

```diff
@@ -0,0 +1,33 @@
+package software.amazon.awssdk.core.exception;
```

Comment:
copyright header?

Reply:
Clearly skipping checkstyle. If only we had a checklist for this...

Reply:
Fixed.